src/assets/appconfig.json

```diff
@@ -9,7 +9,9 @@
     "clientId": "UVGIL27XqwGsJwsTfEyz50c1qg3Ll1p6",
     "authorizationParams": {
       "connection": "google-oauth2"
-    }
+    },
+    "cacheLocation": "localstorage",
```

Comment:
Does BE need to define this for deploys?

Reply:
yep